api/src/main/java/org/geysermc/geyser/api/bedrock/camera/CameraData.java

```diff
@@ -177,4 +177,12 @@ public interface CameraData {
      * @return an unmodifiable view of all currently hidden {@link GuiElement}s
      */
     @NonNull Set<GuiElement> hiddenElements();
+
+    /**
+     * Displays a notification toast for this connection.
+     *
+     * @param title the title of the toast.
+     * @param content the content of the toast.
+     */
+    void sendToast(String title, String content);
```

Comment:
please remove the periods from the end of these param tags phrases. also add nonnull annotations to the params themselves